src/components/SkillCardList/SkillCardList.js

```diff
@@ -85,7 +85,7 @@ class SkillCardList extends Component {
                 '/' +
                 skill.group +
                 '/' +
-                skill_name.toLowerCase().replace(/ /g, '_') +
+                skill.skill_tag +
```

Comment:
Isn't it required to convert this to lowercase as well? For example, the skill_tag for the Skill "Antonyms" is `Antonyms`, which should be converter to lowercase like before. Please check. Thanks :-)

Reply:
There is no hard and fast to change it to lowercase. It causes many inconsistencies on multiple features like rating, feedback, etc, because the iOS and Android clients have implemented in that way. 
For ref - Check feedback of this skill and see the difference
* https://pr-932-fossasia-susi-skill-cms.surge.sh/Social/Farming/en.
* https://skills.susi.ai/Social/Farming/en